src/Installation/AddDocker.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Aero\Cli\Installation;
+
+use Aero\Cli\InstallStep;
+use Symfony\Component\Process\Process;
+
+class AddDocker extends InstallStep
+{
+    /**
+     * Run the installation helper.
+     *
+     * @return void
+     */
+    public function install()
+    {
+        $this->command->output->write('Setting up Docker...');
+
+        $composer = $this->findComposer();
+
+        $commands = [
+            "{$composer} require aerocommerce/docker-environment",
```

Comment:
Might need the `--prefer-dist` option on here for now to make sure it's obtained from the package server rather than github source

Reply:
@timrspratt done